scout-engines/Dockerfile

```diff
@@ -144,67 +144,42 @@ RUN cd wabt-bn128-rolluprs-fastmont-fasthost-superops && make clang-release
 #RUN cd wabt-bn128-rolluprs-fastmont-slowhost && make clang-release
 
 
-# wabt branch for wasmsnark bls12
-RUN git clone --recursive --single-branch --branch bls12-bignums-fastmont-superops https://github.com/ewasm/wabt.git wabt-bls12-fastmont-fasthost-superops
```

Comment:
This seems to have dropped the additions from @cdetrio ?

Reply:
Yea... I'll rebase and add them.